### PR TITLE
Add nuspec file for System.Time library

### PR DESCRIPTION
**Pull request by tarekgh**
This will allow building a nuget package for System.Time library

### CODE REVIEW

File: nuget/System.Time.nuspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Date, TimeOfDay</description>
+    <releaseNotes>Initial package</releaseNotes>
+    <copyright> Microsoft Corporation.  All rights reserved.</copyright>
+    <tags>Date and Time helper classes</tags>
+  </metadata>
+  <files>
+    <file src="..\bin\Windows_NT.AnyCPU.Release\System.Time\System.Time.dll" target="lib/portable-net45+win8+wpa81+aspnetcore50"/>
+  </files>
+</package>
```
Comment (KrzysztofCwalina) [inline nuget/System.Time.nuspec]: I think you need to add the dependencies here. See how it's done in the System.Text.Formatting (which BTW is just an example)

Comment (tarekgh) [inline nuget/System.Time.nuspec]: the dependencies mentioned in System.Text.Formatting are only what is in corefxlab libraries which we don't have any of such dependency in System.Time. 
Looking at other nuspec System.CommandLine.nuspec and I am seeing no dependencies are mentioned there. 
I'll look at that to know if I need to specify the dependencies or not

Comment (KrzysztofCwalina) [inline nuget/System.Time.nuspec]: These are the dependecies I think you need:

    "System.Diagnostics.Contracts": "4.0.0-beta-*",
    "System.Diagnostics.Debug": "4.0.10-beta-*",
    "System.Globalization": "4.0.10",
    "System.Text.RegularExpressions": "4.0.10",
    "System.Xml.ReaderWriter": "4.0.10"

**Comment by tarekgh** [comment]
@KrzysztofCwalina  @socket  could you please have another quick lock?

**Comment by sokket** [comment]
LGTM

**Comment by KrzysztofCwalina** [comment]
looks good

**Pull request merged**